### DISCUSSION TITLE
Pip installation missing yaw_optimization?

### DISCUSSION THREAD

**Discussion by mr483** [NatLabRockies/floris#360] [Q&A] [answered]
I have installed FLORIS 3.0 via pip, but my install seems to be missing the "yaw_optimization" module which should be located at floris.tools.optimization. I noticed this when the import failed while running example 09. When checking the source code, I notice the yaw_optimization folder is missing. I tried to reinstall on a separate environment and experience the same issue. I noticed this folder seems to not have an __init__ file in it, which may be causing the issue.

**Comment by rafmudaf** [comment, marked as answer]
Yep @mr483 this looks like a bug. Thanks for reporting it. I'm creating an issue for this and we'll work on fixing it asap.
